apps/frontend/core-dapp/src/components/Menu.tsx

```diff
@@ -12,8 +13,30 @@ export const MenuItem: React.FC<MenuItemProps> = (props) => (
 
 type Props = {
   size?: 'sm' | 'md' | 'lg'
+  keepBackgroundColorOnHover?: boolean
 }
 
+const keepBackgroundOnHover = css`
+  .ant-dropdown-menu-item-active {
+    background-color: ${({ theme }): string => theme.color.neutral10};;
+    * {
+      color: ${({ theme }): string => theme.color.neutral3};
+      font-weight: ${({ theme }): number => theme.fontWeight.semiBold};
+    }
+    :hover {
+      * {
+        color: ${({ theme }): string => theme.color.primary};
+        font-weight: ${({ theme }): number => theme.fontWeight.medium};
+      }
+    }
+`
```

Comment:
Missing one closing curly bracket here
```suggestion
const keepBackgroundOnHover = css`
  .ant-dropdown-menu-item-active {
    background-color: ${({ theme }): string => theme.color.neutral10};;
    * {
      color: ${({ theme }): string => theme.color.neutral3};
      font-weight: ${({ theme }): number => theme.fontWeight.semiBold};
    }
    :hover {
      * {
        color: ${({ theme }): string => theme.color.primary};
        font-weight: ${({ theme }): number => theme.fontWeight.medium};
      }
    }
    }
`
```